roq-deployment/development/HCM.log

```diff
@@ -0,0 +1,8 @@
+Starting the local host configuration manager with 1
```

Comment:
This file should not have been committed. Please remove it and add it to the .gitignore file if necessary.